frontend/src/components/Diff/Diff.tsx

```diff
@@ -185,59 +185,86 @@ function DiffBody({ diff, fontSize }: { diff: api.DiffOutput, fontSize: number |
     </div>
 }
 
+function ThreeWayToggleButton({ enabled, setEnabled }: { enabled: boolean, setEnabled: (enabled: boolean) => void }) {
+    return <button
+        className={styles.threeWayToggle}
+        onClick={() => {
+            setEnabled(!enabled)
+        }}
+        title={enabled ? "Disable three-way diffing" : "Enable three-way diffing"}
+    >
+        {enabled ? "3" : "2"}
```

Comment:
Possibly needs some iteration, like an icon or something. On mobile you can't ever see the title

Reply:
I would love some help with this, my design skills are terrible.

Reply:
Maybe we use this icon? https://primer.style/foundations/icons/git-merge-16